tests/test_variant_coordinates.py

```diff
@@ -1,24 +1,48 @@
-from opentargets_pharmgkb.variant_coordinates import get_coordinates_for_clinical_annotation
+import os
 
+import pytest
 
-def test_get_coordinates_single_alt():
-    # rs4343: G/A
-    # When there's only one alt we don't need the genotypes
-    assert get_coordinates_for_clinical_annotation('rs4343', []) == '17_63488670_G_A'
+from opentargets_pharmgkb.variant_coordinates import Fasta
 
 
-def test_get_coordinates_multiple_alts():
-    # rs4659982: T/A/C
-    assert get_coordinates_for_clinical_annotation('rs4659982', ['TT', 'TA', 'AA']) == '1_240566955_T_A'
-    # Use of different alt allele generates a different identifier
-    assert get_coordinates_for_clinical_annotation('rs4659982', ['TT', 'TC', 'CC']) == '1_240566955_T_C'
-    # Only two alleles in genotypes, but neither is reference which is always included
-    assert get_coordinates_for_clinical_annotation('rs4659982', ['AA', 'CC']) == None
+@pytest.fixture
+def fasta():
+    fasta_path = os.path.join(os.path.dirname(__file__), 'resources', 'chr21.fa')
+    return Fasta(fasta_path)
 
 
-def test_get_coordinates_deletion():
-    # rs70991108: -/TCGCGCGTCCCGCCCAGGT/TGGCGCCTCCCGCCCAGGT/TGGCGCGTCCCGCCCAGGT
-    assert get_coordinates_for_clinical_annotation(
-        'rs70991108',
+def test_get_coordinates(fasta: Fasta):
+    assert fasta.get_coordinates_for_clinical_annotation(
+        'rs1051266',
+        'NC_000021.9:33341701',
+        ['GG', 'GT', 'TT']) == '21_33341701_G_T'
+    # Use of different alt allele in genotypes generates a different identifier
+    assert fasta.get_coordinates_for_clinical_annotation(
+        'rs1051266',
+        'NC_000021.9:33341701',
+        ['GG', 'GA', 'AA']) == '21_33341701_G_A'
```

Comment:
These genotypes and coordinates are made up, right?

Reply:
Yes that's right